README.md

```diff
@@ -62,12 +77,22 @@ Parsing errors will be printed in terminal as the files are loaded.
 
 This software is primarily based on a
 [similar project](https://sourceforge.net/projects/nuclei/)
-by M.A.Nagel.
+by M.A.Nagel, as it can be seen in the git history.
+
+A description of the original Nuclei's functionality and results obtained using its search method was published in Nuclear Instruments and Methods in Physics Research, Section A:
+
+M. Nagl, et al., NIM A 726 (2013), 17-30
+[doi:10.1016/j.nima.2013.05.045]
```

Comment:
maybe better on the same line as reference, and could also incorporate a link via DOI?